### DISCUSSION TITLE
flowbite-angular for Angular 17.3.11

### DISCUSSION THREAD

**Discussion by ShadyManu** [themesberg/flowbite-angular#41] [Q&A]
Hello everyone, I'm trying to install flowbite on my angular 17 app, and when I do
npm i flowbite-angular
I get unable to resolve dependency tree.. the latest version on npmjs site is the 0.0.2
[flowbite-angular](https://www.npmjs.com/package/flowbite-angular?activeTab=versions)
because the 0.0.2 version requires Angular 13.3.0, but from your [GitHub Changelog](https://github.com/themesberg/flowbite-angular/blob/main/CHANGELOG.md) I can see that there should be the 0.0.3 version which include support for Angular 17.. How can I install this 0.0.3 version on my app?

**Comment by ShadyManu** [comment]
Nothing? So sad because I really like this library

**Reply by m4dzkdev** [reply]
same problem bro, pls help!

**Reply by pamgaiguer** [reply]
same thing here! with angular 17 and flowbite 2.4.1

**Reply by ShadyManu** [reply]
There is no solution right now. The only solution is use the non angular version. On their discord they said they've been working on that for months, but it's still not available. Not much we can do now. I'm using the non angular version

**Comment by MGREMY** [comment]
@ShadyManu @m4dzkdev  @pamgaiguer Hello, hmm, 4 months late :/
There is no solution for the moment because we are working on a huge update since nearly 1 year, it should come (hopefully) in 2 months, maybe 3 if we have some issues that we didn't see before

For the 0.0.3 in the changelog it's my fault sorry, I was just messing around the repo and created a changelog... :/ So sorry about that.
If you are on the flowbite's discord ([here is the link](https://discord.gg/tJup9qvz)), you can read all the update on this project in the "angular" room, I'll notify everyone here when a new release will be coming.

I hope you're not too sad about my false changelog/version, and hope to see you in discord soon, and as always, if you want to help and the project you are welcome ! :D
